example/spec/parallel/testAPI/sqsRuleSpec.js

```diff
@@ -92,6 +93,7 @@ async function cleanUp() {
   });
 
   // TODO there may be another execution to delete here
+  await waitForCompletedExecution(executionArn);
```

Comment:
We should probably make sure the feature branch is waiting for the *api* execution call here.